src/lib/webhook.ts

```diff
@@ -0,0 +1,19 @@
+import axios from "axios"
+import * as process from "process"
+
+export async function sendWebhook<T>(
+  aggregator: string,
+  event: string,
+  data: T,
+) {
+  const url = [
+    process.env.FLOWCORE_WEBHOOK_BASEURL,
+    "event",
+    process.env.FLOWCORE_TENANT,
+    process.env.FLOWCORE_DATACORE,
+    aggregator,
+    event,
+  ].join("/")
+  console.log("webhook", url)
```

Comment:
I'm assuming this console.log can be binned :P

Reply:
one minor optimization we can consider later (if we ever need to). Is to pre-generate the url up to the aggregator point, since they will never change unless the app is re-deployed)